Keas.Mvc/Controllers/Api/PeopleController.cs

```diff
@@ -174,6 +174,20 @@ public async Task<IActionResult> SearchUsers(string searchTerm)
                 User = user,
                 Title = await _identityService.GetTitle(user.Iam)
             };
+            var supervisor = await _identityService.GetIamSupervisor(user.Iam);
+            if (supervisor != null)
+            {
+                var superPerson = await _context.People
+                    .Where(x => x.Team.Slug == Team && x.UserId == supervisor.Id)
+                    .Include(x => x.Supervisor)
+                    .IgnoreQueryFilters()
+                    .FirstOrDefaultAsync();
```

Comment:
Why the include? Why not just do the where and then project out `select()` just the Id that you need.  Much more efficient query for the same result.

Reply:
That was a copy/paste mistake. Fixed